src/components/navbar/Navbar.module.css

```diff
@@ -0,0 +1,42 @@
+.logo {
+  width: 3rem;
+  height: 3rem;
```

Comment:
- [ ] Kindly adjust the height of the Navbar logo to match the given design, the logo looks a bit squeezed.
![FireShot Capture 170 - React App - localhost](https://user-images.githubusercontent.com/46851023/151183709-ab4aaedd-12de-4fcd-b9d1-0d23e3bb63cd.png)

Reply:
May I ask what browser you are using? @ZeenatLawal

Reply:
@akucintavalent I use chrome